### DISCUSSION TITLE
Feature request: per-world claim limits

### DISCUSSION THREAD

**Discussion by SlimeDog** [GriefPrevention/GriefPrevention#267] [Ideas]
### Spigot/Craftbukkit/Paper and GriefPrevention version:
Spigot 1.12.2
GP 16.8

### What behaviour is observed:
GP claim limits apply to all worlds together.

### What behaviour is requested:
Set claim limits per-world. Use case: I have a creative world and four survival worlds on one server. I would like to grant large(r) limits in the creative world, and separate limits in each of the survival worlds.

### Plugin list (if applicable):
Multiverse-Core

**Comment by RoboMWM** [comment]
Unsure what you mean by limits, but very likely a duplicate of #141. Please close if this is the case, otherwise clarify. Thanks!

**Comment by SlimeDog** [comment]
I don't read this the same as #141. I'm looking for separate Claim Blocks per-world. The idea is to allow higher limits in some worlds than in others, and to not share across worlds.

At present, configuration specifies
```
GriefPrevention:
  Claims:
    Max Accrued Claim Blocks:
      Default: 2000
```

I want to create separate limits in each world. For example, in (creative) world, I want to allow (say) 5000 blocks maximum, but in (survival) world_xyzzy, I want to allow only 500 blocks maximum, and in (survival) world_abcde, I want to allow only 400 blocks maximum. So configuration would have to support something like
```
GriefPrevention:
  Claims:
    Max Accrued Claim Blocks:
      Default:
        world: 5000
        world_abcde: 400
        world_xyzzy: 500
```

If this is supported, it is important that the 5000 available in world not be transferred to or available in world_abcde, for example.
